tensorflow_addons/activations/rrelu_test.py

```diff
@@ -24,54 +24,58 @@
 from tensorflow_addons.activations import rrelu
 from tensorflow_addons.utils import test_utils
 
-
-def _ref_rrelu(x, lower, upper):
-    return tf.where(x >= 0, x, (lower + upper) * x / 2)
+SEED = 111111
 
 
 @test_utils.run_all_in_graph_and_eager_modes
 class RreluTest(tf.test.TestCase, parameterized.TestCase):
     @parameterized.named_parameters(("float16", np.float16),
                                     ("float32", np.float32),
                                     ("float64", np.float64))
-    @tf.function
     def test_rrelu(self, dtype):
         x = tf.constant([-2.0, -1.0, 0.0, 1.0, 2.0], dtype=dtype)
         lower = 0.1
         upper = 0.2
-        result = rrelu(x, lower, upper, training=False)
-        expect_result = _ref_rrelu(x, lower, upper)
-        self.assertAllCloseAccordingToType(result, expect_result)
+
+        training_results = {
+            np.float16: [-0.288330078, -0.124206543, 0, 1, 2],
+            np.float32: [-0.26851666, -0.116421416, 0, 1, 2],
+            np.float64: [-0.3481333923206531, -0.17150176242558851, 0, 1, 2],
+        }
+        for training in [True, False]:
+            with self.subTest(training=training):
+                tf.random.set_seed(SEED)
```

Comment:
Is it necessary?

Reply:
Yes, according to https://github.com/tensorflow/tensorflow/issues/33034#issuecomment-539752847